File: examples/LINUX/benchmarks/programs/pipes.py
import subprocess

s = "This is a test string.\n"
p = subprocess.Popen(['sh', '-c', 'cat > /tmp/biplan-benchmark-pipes.txt'], stdin=subprocess.PIPE)
for i in range(1000):
    p.stdin.write(s.encode())
p.stdin.close()
p.wait()

File: examples/LINUX/benchmarks/programs/pipes.php
<?php

$s = "This is a test string.\n";
$pipe = popen('sh -c "cat > /tmp/biplan-benchmark-pipes.txt"', 'w');
for($i = 0; $i < 1000; $i++)
  fwrite($pipe, $s);
pclose($pipe);
